dataprofiler/tests/profilers/test_column_profile_compilers.py

```diff
@@ -411,9 +412,144 @@ def test_base(self, *mocks):
             output_profile['statistics']['vocab'] = \
                 sorted(output_profile['statistics']['vocab'])
 
-        self.maxDiff = None
         self.assertDictEqual(expected_dict, output_profile)
 
+    @mock.patch('dataprofiler.profilers.unstructured_labeler_profile.'
+                'DataLabeler')
+    @mock.patch('dataprofiler.profilers.unstructured_labeler_profile.'
+                'CharPostprocessor')
+    def test_compiler_stats_diff(self, *mocks):
+        data1 = pd.Series(['Hello Hello', 'This is a test grant'])
+        data2 = pd.Series(['This is unknown', 'my name grant', '9', '9'])
+
+        # Test normal diff
+        compiler1 = col_pro_compilers.UnstructuredCompiler(data1)
+        compiler2 = col_pro_compilers.UnstructuredCompiler(data2)
+        labeler_1 = compiler1._profiles["data_labeler"]
+        labeler_2 = compiler2._profiles["data_labeler"]
+
+        labeler_1.char_sample_size = 20
+        labeler_1.word_sample_size = 15
+        labeler_1.entity_counts["postprocess_char_level"]["UNKNOWN"] = 5
+        labeler_1.entity_counts["postprocess_char_level"]["TEST"] = 10
+        labeler_1.entity_counts["postprocess_char_level"]["UNIQUE1"] = 5
+        labeler_1.entity_counts["true_char_level"]["UNKNOWN"] = 4
+        labeler_1.entity_counts["true_char_level"]["TEST"] = 8
+        labeler_1.entity_counts["true_char_level"]["UNIQUE1"] = 8
+        labeler_1.entity_counts["word_level"]["UNKNOWN"] = 5
+        labeler_1.entity_counts["word_level"]["TEST"] = 5
+        labeler_1.entity_counts["word_level"]["UNIQUE1"] = 5
```

Comment:
not going to block, but any reason not a dict?

Reply:
fixed